test/validators.test.js

```diff
@@ -14463,54 +14463,32 @@ describe('Validators', () => {
       ],
     });
   });
-  it('should validate mailto URI', () => {
```

Comment:
Don't remove existing tests

Reply:
I just copied the existing test for the format and modified it according to my function.

Reply:
@WikiRik can you check I modified the code and fixed the bugs!